engine-tests/src/test/java/org/terasology/documentation/apiScraper/ApiComparator.java

```diff
@@ -0,0 +1,289 @@
+        System.out.println("Checking Class Addition and Deletion");
+        for (String className : originalApi.keySet()) {
+            if (!newApi.containsKey(className)) {
+                System.out.println("MAJOR INCREASE, DELETION OF " + className);
+            }
+        }
+
+        for (String className : newApi.keySet()) {
+            if (!originalApi.containsKey(className)) {
+                System.out.println("MINOR INCREASE, ADDITION OF " + className);
+            }
+        }
+    }
+
+    /**
+     * Checks creation and deletion of methods, as well as existing method changes
+     * @param originalApi the original api generated from ORIGINAL_API_FILE
+     * @param newApi the new apí generated from NEW_API_FILE
+     */
+    private static void checkMethodChanges(Map<String, Collection<ApiMethod>> originalApi,
+                                           Map<String, Collection<ApiMethod>> newApi) {
+        System.out.println("Checking Method Changes");
+        Collection<ApiMethod> originalMethods;
+        Collection<ApiMethod> newMethods;
+        for (String className : originalApi.keySet()) {
+
+            originalMethods = originalApi.get(className);
+            newMethods = newApi.get(className);
+            if (newMethods == null) {
+                continue;
+            }
+            checkMethodDeletion(originalMethods, newMethods);
+
+            for (ApiMethod method2 : newMethods) {
+
+                boolean found = false; // if found, the method is an existing one or a new overloaded method
+                for (ApiMethod method1 : originalMethods) {
+                    if (method1.getName().equals(method2.getName())) {
+
+                        ApiMethod auxMethod = getMethodWithSameNameAndParameters(method2, originalMethods);
+                        if (auxMethod.getName().equals("")) {
+                            ApiMethod auxMethod2 = getMethodWithSameNameAndParameters(method1, newMethods);
+                            if (auxMethod2.getName().equals("")) {
+                                checkMethodIncrease(method1, method2);
+                            } else if (isInterfaceOrAbstract(method2.getClassName())) {
+                                System.out.println("MINOR INCREASE, NEW OVERLOADED METHOD " + method2.getName() +
+                                        " ON " + method2.getClassName() + "\nNEW PARAMETERS: " + method2.getParametersType());
+                                System.out.println("=================================================================");
+                            }
+
+                        } else {
+                            checkMethodIncrease(auxMethod, method2);
+
+                        }
+                        found = true;
+                    }
+                }
+                if (!found && isInterfaceOrAbstract(method2.getClassName())) {
```

Comment:
This misses a new public method in a non-abstract class. That should count as a minor change :-)

I'm not sure if that was done previously and we're just going back and forth on what's included, or if it was legit missed